src/pages/index.html

```diff
@@ -25,52 +25,115 @@
     <div class="row">
       <div class="col-7">a</div>
       <div class="col-5">b
-        <div class="card">
+        <div class="card card__content">
```

Comment:
Don't mix `card` and `card__content` classes.

As you can see, `card__content` is the *element* of the `card` block. Instead, *nest* them:

```html
<div class="card">
  <div class="card__content">
    <!-- children -->
  </div>
</div>
```